Update VRL functions list in upgrade_worker.md (#37627)
DataDog/documentation: content/en/observability_pipelines/guide/upgrade_worker.md

#### New features

> - The Custom Processor now supports the`to_entries` and `from_entries` functions for converting between objects and arrays of key-value pairs (jq-style).

Reviewer: ```suggestion
- The Custom Processor now supports the `to_entries` and `from_entries` functions for converting between objects and arrays of key-value pairs (jq-style).
```

> - The Custom Processor functions`to_entries` and `from_entries` are now available for converting between objects and arrays of key-value pairs (jq-style).

Author: ```suggestion
- The Custom Processor now supports the`to_entries` and `from_entries` functions for converting between objects and arrays of key-value pairs (jq-style).
```

> - The `to_entries` and `from_entries` VRL functions are now available for the Custom Processor for converting between objects and arrays of key-value pairs (jq-style).

Author: ```suggestion
- The Custom Processor functions`to_entries` and `from_entries` are now available for converting between objects and arrays of key-value pairs (jq-style).
```

> - The following VRL functions are now available for the Custom Processor: `to_entries` and `from_entries` for converting between objects and arrays of key-value pairs (jq-style).

Author: ```suggestion
- The `to_entries` and `from_entries` VRL functions are now available for the Custom Processor for converting between objects and arrays of key-value pairs (jq-style).
```
